packages/teams-js/src/public/calendar.ts

```diff
@@ -54,15 +65,24 @@ export namespace calendar {
     return ensureInitialized(runtime) && runtime.supports.calendar ? true : false;
   }
 
+  /** Open calendar item parameters. */
   export interface OpenCalendarItemParams {
+    /** A string representing the unique ID of the calendar item to be opened. */
     itemId: string;
   }
 
+  /** Compose meeting parameters */
+
   export interface ComposeMeetingParams {
+    /** An array of email addresses of the attendees to invite to the meeting. */
```

Comment:
>An array of email addresses of the attendees to invite to the meeting. [](http://example.com/codeflow?start=8&length=70)

If this is literally only ever e-mail addresses, should we look to be more specific with the type than just `string` ? Could there be multiple formats for the e-mail address or is it only ever person@example.com format?

Reply:
Good point! Does this looks good?

```
An array string that represents the identifier of an attendee

Reply:
Similar to previous comments is there any guidance we can give about what these identifiers are?

Reply:
Updated the comment.